pytentiostat/plotter.py

```diff
@@ -3,6 +3,18 @@
 
 
 def plot_initializer(config_data):
+    """
+    Initializes plot based on experiment type and parameters in the config file.
+
+    Parameters
+    ----------
+    config_data : dict
+        Parameters from the config file
```

Comment:
The numpy style would prepend a "the" here.  I don't really know why, but it actually really works to make things more intelligible in my opinion, but personally I strongly recommend it.  It kind of helps my brain's internal conversation when I am writing these too, so  it kind of goes "oh, config_data, what was that again?, oh yes, it is a dict that contains the parameters from the config file".  so I then write "The parameters from the config file"

Reply:
I agree, sounds good now!